OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs

```diff
@@ -120,7 +124,7 @@ IEnumerable<IRenderable> IRenderAnnotationsWhenSelected.RenderAnnotations(Actor
 
 			// Reverse draw order so target markers are drawn on top of the next line
 			renderableCache.Reverse();
-			return renderableCache;
+			return renderableCache.Count == 0 ? Enumerable.Empty<IRenderable>() : renderableCache.ToArray();
```

Comment:
Wouldn't it make sense to omit to `Reverse` call if there are no elements? (It also looks more readable.)
```
if (renderableCache.Count == 0)
	return Enumerable.Empty<IRenderable>();

// Reverse draw order so target markers are drawn on top of the next line
return renderableCache.Reverse().ToArray();
```

Wouldn't it also make sense to replace the `Any` calls with `Count` inside the for loop?

Reply:
Definitely more readable - have adjusted and applied the Count change too.